build/props/common.props

```diff
@@ -45,6 +52,9 @@
     <!-- SourceLink -->
     <PropertyGroup Condition="'$(SilkSourceLinkExempt)' == ''">
         <DebugType>portable</DebugType>
+        <EmbedAllSources>true</EmbedAllSources>
```

Comment:
This embeds in the snupkg instead of the nupkg, which are massive anyway. I'm not as concerned about the symbols packages, mainly because they're only downloaded if you need them (e.g. Ctrl+Click, debugging, etc)

Reply:
I see - I must've misunderstood the docs.

Reply:
You're absolutely correct that it we're using the embedded debug format (which embeds symbols directly into the dll) that our packages would be HUGE